exchanges/collateral/collateral.go

```diff
@@ -0,0 +1,70 @@
+package collateral
+
+import (
+	"encoding/json"
+	"strings"
+)
+
+// Valid returns whether the margin type is valid
```

Comment:
margin 👀 and across this package. Also suggestion: margin/collateral can be interchangeable so I wouldn't oppose consolidation of packages.

Reply:
Kinda sorta. I think margin as a package can be expanded into things like margin borrowing, rates and repayment and so I think they can remain separated. However at this current point in time they could be merged. Would you like me to merge them?

Reply:
Nope, it's fine.